ppr-ui/src/components/unitNotes/UnitNoteAdd.vue

```diff
@@ -1,27 +1,36 @@
 <template>
   <div>
     <section id="mhr-unit-note-doc-id" class="mt-10">
-      <h2>Document ID</h2>
-      <p class="mt-2">
-        Enter the 8-digit Document ID number.
-      </p>
-      // Add Document ID placeholder //
+      <DocumentId
+        :documentId="unitNoteDocumentId"
+        :setStoreProperty="setDocumentId"
+        :validate="true"
```

Comment:
Do we want validate true here? I assume this will be updated during a validation ticket.

Reply:
Good catch, slipped in by mistake.